Userland/DevTools/SQLStudio/MainWidget.cpp

```diff
@@ -135,6 +137,10 @@ MainWidget::MainWidget()
         update_editor_actions(editor);
     });
 
+    m_run_script_action = GUI::Action::create("Run script", { Mod_Alt, Key_F9 }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/go-forward.png").release_value_but_fixme_should_propagate_errors(), [&](auto&) {
```

Comment:
I think the play icon would be more appropriate, Hack Studio uses a similar icon for its run action